packages/server/postgres/migrations/1717685812677_addMeetingTemplateEmbedding.ts

```diff
@@ -0,0 +1,37 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+  DO $$
+  BEGIN
+    ALTER TYPE "EmbeddingsObjectTypeEnum" ADD VALUE IF NOT EXISTS 'meetingTemplate';
+  END $$;
+  `)
+  await client.end()
+}
+
+export async function down() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+  DO $$
+  BEGIN
+    DELETE FROM "EmbeddingsMetadata" WHERE "objectType" = 'meetingTemplate';
```

Comment:
+1 would using kysely work here instead?

Reply:
I think it would. I'm not sure what I would gain in that case. I guess typesafety after the up migration was run, but that seems a bit odd 🤔 
Any particular reason we should use kysely in migrations?

Reply:
I don't see any examples of this elsewhere and I have some concerns regarding the typecheck. The typecheck would fail before it was run. In theory we could end up with migrations which never pass typecheck if the types differ between `up` and `down` too much.